Web/develop/documentation.rst

```diff
@@ -62,4 +67,120 @@ Building the documentation on Linux
+
+Set up a virtual python environment and name it something like CP27 (that's what is used in the examples below).  This virtual environment will contain all the modules needed to build the CoolProp documentation.  Setting up a virtual environment is a very simple thing to do in the Anaconda Navigator (Graphical Interface), but you can also set up the environment using the following commands in a command line window::
+
+    conda create --name CP27 python=2.7
+    activate CP27 # Will cause command prompt to be prefixed with (CP27)
+	
+.. note::
+
+   Any instructions below that take place on the Windows are assumed to be in the virtual environment created here.  For example, when opening a Windows command prompt (cmd), *activate CP27* first before issing any other commands.
+
+**Install Required Python Modules**
+
+Most of the following modules can be installed from the Anaconda Navigator, and it is much simpler to do so.  Even if the module is already in the Anaconda environment, it is best to update the packages listed, making sure to get the latest Windows 64-bit versions.  However, some of the requisite modules are not in the Conda library and have to be installed from the command line using `pip install`.  These will be designated as such.  The versions of each module that have been tested for generating the docs on Windows are shown in parentheses below. 
+
+ - setuptools       (27.2.0)
+ - cython           (0.25.2)
+ - numpy            (1.12.1)
+ - scipy            (0.19.0)
+ - matplotlib       (2.0.2)
+ - python-dateutil  (2.6.0)
+ - pytz             (2017.2)
+ - pandas           (0.20.1)
+ - sphinx           (1.5.6)
+ - ipython          (5.3.0)
+ - wxpython         (3.0)  
+
+The following modules will need to be installed using *pip install*.
+
+ - sphinxcontrib-bibtex    (6.3.5)  
+ - sphinxcontrib-doxylink  (1.3)    
+ - sphinxcontrib-napoleon  (0.6.1)  
+ - cloud-sptheme           (1.9.4)
+
+.. note::
+
+   Errors may occur during the install of some of the above Python modules or while making the html docs.  Make sure that all of the above Python modules are the most recent versions using pip install --upgrade.  There are known issues with the default sphinx version installed with Anaconda on Windows.
+   
+If updating the dev version of the documentation (i.e. the latest dev source files are loaded), the dev version of the CoolProp module will need to be installed.  This ensures that any example code will run properly with the latest CoolProp functions.::
+
+    pip install -vvv --pre
+     --trusted-host www.coolprop.dreamhosters.com
+     --find-links http://www.coolprop.dreamhosters.com/binaries/Python/ -U 
+     --force-reinstall CoolProp
+   
+
+**Run Setup Scripts**
+
+There are a number of setup scripts that have to be run to generate dynamic content for the web documentation.  To run them, open a windows command prompt (cmd) and cd to the CoolProp\\Web\\scripts directory.  Then run the following scripts::
```

Comment:
You might want to refer to the script runner at https://github.com/CoolProp/CoolProp/blob/master/Web/scripts/__init__.py#L107